### PR TITLE
File Changes

### PR DESCRIPTION
**Pull request by JustCat80**
Deprecates the file argument within createMessage for content.file to be consistent with executeWebhook
Also adds support for editing files within a message with editMessage

### CODE REVIEW

File: lib/Client.js [hunk 1]
```diff
@@ -609,12 +612,16 @@ class Client extends EventEmitter {
...
-    * @arg {Object | Array<Object>} [file] A file object (or an Array of them)
+    * @arg {Object | Array<Object>} [file] [DEPRECATED] A file object (or an Array of them). Use `content.file` instead
     * @arg {Buffer} file.file A buffer containing file data
     * @arg {String} file.name What to name the file
     * @returns {Promise<Message>}
     */
     createMessage(channelID, content, file) {
+        if(file) {
+            this.emit("warn", "[DEPRECATED] File as a seperate argument is deprecated. Use content.file instead");
+            content.file = file;
```
Comment (ghost) [inline lib/Client.js]: Cannot create property "file" on a string.

Comment (JustCat80) [reply lib/Client.js]: Good catch yea I'm stupid

Comment (JustCat80) [reply lib/Client.js]: Fixed

**Comment by JustCat80** [comment]
Recreating this uwu

**Pull request closed without merging**